README.md

```diff
@@ -222,6 +223,187 @@ There are number of techniques you can use to ensure that access token expiry is
+
+These changes are added as follows:
+
+```.objc
+// TVOCall.h
+typedef NS_ENUM(NSUInteger, TVOCallState) {
+    TVOCallStateConnecting = 0, ///< The Call is connecting.
+    TVOCallStateRinging,        ///< The Call is ringing.
+    TVOCallStateConnected,      ///< The Call is connected.
+    TVOCallStateDisconnected    ///< The Call is disconnected.
+};
+
+// TVOCallDelegate.h
+@protocol TVOCallDelegate
+
+@optional
+- (void)callDidStartRinging:(nonnull TVOCall *)call;
+
+@end
+```
+
+### Media Establishment & Connectivity
+The Voice iOS 3.X SDK uses WebRTC. The exchange of real-time media requires the use of Interactive Connectivity Establishment(ICE) to establish a media connection between the client and the media server. In some network environments where network access is restricted it may be necessary to provide ICE servers to establish a media connection. We reccomend using the [Network Traversal Service (NTS)](https://www.twilio.com/stun-turn) to obtain ICE servers. ICE servers can be provided when making or accepting a call by passing them into `TVOConnectOptions` or `TVOAcceptOptions` in the following way:
+
+```.swift
+var iceServers: Array<TVOIceServer> = Array()
+let iceServer1: TVOIceServer = TVOIceServer(urlString: "stun:global.stun.twilio.com:3478?transport=udp",
+                                            username: "",
+                                            password: "")
+iceServers.append(iceServer1)
+
+let iceServer2: TVOIceServer = TVOIceServer(urlString: "turn:global.turn.twilio.com:3478?transport=udp",
+                                            username: "TURN_USERNAME",
+                                            password: "TURN_PASSWORD")
+iceServers.append(iceServer2)
+
+let iceOptions: TVOIceOptions = TVOIceOptions { (builder) in
+    builder.servers = iceServers
+}
+
+// Specify ICE options in the builder
+let connectOptions: TVOConnectOptions = TVOConnectOptions(accessToken: accessToken) { (builder) in
+    builder.iceOptions = iceOptions
+}
+
+let acceptOptions: TVOAcceptOptions = TVOAcceptOptions(callInvite: callInvite!) { (builder) in
+    builder.iceOptions = iceOptions
+}
+```
+
+### Audio Device

```

Comment:
This section needs to be re written. I would devide it in to following -

## Audio Device

### TVIDefaultAudioDevice

### CallKit

### TVIAudioDevice

Reply:
This section should also talk about deprecation of `CallKit` category from `TwilioVoice`.

Reply:
I think the guide copied from the changelog does cover both default audio device and the TVOAudioDevice protocol and clearly that the `CallKitIntegration` category has been deprecated.